src/components/select-field/SelectField.scss

```diff
@@ -2,6 +2,8 @@
 
 .bdl-SelectField,
 .select-field {
+    outline: none;
```

Comment:
I'm thinking we don't want to do this. There's been a fair bit of effort lately to fix the a11y of our various components and to undo this sort of CSS. Is this needed for something specific?